packages/superset-ui-chart/src/components/SuperChart.tsx

```diff
@@ -116,17 +116,21 @@ export default class SuperChart extends React.PureComponent<SuperChartProps, {}>
     );
   }
 
-  processChartProps: (input: {
-    chartProps: ChartProps;
-    preTransformProps?: PreTransformProps;
-    transformProps?: TransformProps;
-    postTransformProps?: PostTransformProps;
-  }) => any;
-
-  createLoadableRenderer: (input: {
-    chartType: string;
-    overrideTransformProps?: TransformProps;
-  }) => LoadableRenderer<RenderProps, LoadedModules> | (() => null);
+  processChartProps: (
```

Comment:
@kristw this is odd -- my linter locally would convert it to this formatting when running `git commit` but CI doesn't like this new format. 🤔 Do you guys have this issue this locally?

Reply:
Umm, I don't have this issue.

Reply:
I did encounter this issue a couple times, was inconsistent tho 🤔